website/views/options/social/index.jade

```diff
@@ -113,3 +113,17 @@ script(type='text/ng-template', id='partials/options.social.html')
   .tab-content
     .tab-pane.active
       div(ui-view)
+
+script(type='text/ng-template', id='modals/copyChatToDo.html')
```

Comment:
Seems that only one "mode" is possible - see http://angular-ui.github.io/bootstrap/#/tooltip , couldn't find anything helpful online (to have both modes)